careers/ladder.md

```diff
@@ -563,15 +563,15 @@ See
 [this document](https://www.notion.so/artsy/Artsy-Tech-Lead-Expanded-Responsibilities-Public-710ee0cd2fce47dcaa722ece80e365b2)
 for an expanded set of Tech Lead Responsibilities at Artsy.
 
-## Management
+## Engineering Management
 
-A manager’s role at Artsy balances few individual contributor responsibilities with more people management
-responsibilities. As an engineering manager grows at Artsy, any code they write is expected to shift from
-delivering direct value on project progress to having a multiplying effect by enabling their team to do better
-work. As such, an Engineering Manager has direct reports and their performance is evaluated by the scale and impact
-of their team considerably more than the direct impact of the code they write.
+A manager’s role at Artsy balances people management responsibilities with team delivery responsibilities. As a
+people manager, they are responsible for hiring, retaining, growing and managing the performance of their reports.
+Additionally, they're expected to significantly impact teams' delivery by providing a strong technical contribution
```

Comment:
Suggested rewording to remove repeated or ambiguous words:

> Additionally, they're expected to significantly impact teams' success through their own technical contribution and/or delivery management.

Personally, I'd also love to clarify what's meant by delivery management. I think we mean many things like team process, tools, release strategy, external coordination and communication.... Is there a concise way to capture this?

Reply:
good call, I updated it.

Reply:
Couldn't find a concise way to definer delivery management, so I went for the verbose way instead. It's still better than not defined at all.